spec/cf_app_discovery/target_configuration_spec.rb

```diff
@@ -71,7 +73,7 @@
         },
       },
       {
-        targets: ["app-1.example.com"],
+        targets: ["route-1.example.com"],
```

Comment:
(argh previous review broke)

This test looks like it hardcodes one single domain (in paas_domain) for all routes. If so, that won't work for custom domains -- it'll only work for cloudapps.digital domains.

Do we need more work to support custom domains? We know that our users need this. (it could be a separate PR if necessary).

Reply:
(after offline chat) we've agreed to do custom domains in a separate PR

Reply:
👍